src/sentry/grouping/fingerprinting/__init__.py

```diff
@@ -466,3 +494,43 @@ def visit_key(self, node, children):
     def visit_quoted_key(self, node, children):
         # leading ! are used to indicate negation. make sure they don't appear.
         return node.match.groups()[0].lstrip("!")
+
+
+def _load_configs():
+    if not CONFIGS_DIR.exists():
+        logger.warning(
+            "Failed to load Fingerprinting Configs, invalid _config_dir: %s",
+            CONFIGS_DIR,
+        )
+
+    configs = {}
+
+    for config_file_path in sorted(CONFIGS_DIR.glob("**/*.txt")):
+        config_name = config_file_path.parent.name
+        configs.setdefault(config_name, [])
+
+        try:
+            with open(config_file_path) as config_file:
+                str_conf = config_file.read().rstrip()
+                configs[config_name].extend(FingerprintingRules.from_config_string(str_conf).rules)
+        except InvalidFingerprintingConfig as ex:
+            logger.warning(
+                "Fingerprinting Config %s Invalid: %s",
```

Comment:
You can use f-strings to help this read a bit better
`f"Fingerprinting Config {config_file_path} Invalid: {ex}"`

Reply:
@armenzg that's is specifically verboten because of ... Sentry :-P

https://github.com/adamchainz/flake8-logging/blob/main/README.rst#log011-avoid-pre-formatting-log-messages